### PR TITLE
feat(slider): allow setting which Pits to show  

### PR DESCRIPTION
**Pull request by Haroenv**
A prop `valuesToShow={[0, 50, 100]}` allows you to specify which percentages to show. The annoying part is that not for every % a Pit is rendered. The values are:

```js
const pitValues = [0, 3, 6, 9, 12, 15, 18, 21, 24, 26, 29, 32, 35, 38, 41, 44, 47, 50, 53, 56, 59, 62, 65, 68, 71, 74, 76, 79, 82, 85, 88, 91, 94, 97, 100];
```

I'm not sure what to make of it.

related to #2276

Only for review since I continued on #2276

### CODE REVIEW

File: src/components/Slider/Pit.js [hunk 1]
```diff
@@ -17,7 +16,7 @@ const Pit = ({style, children}) => {
       }) }
     >
       { shouldDisplayValue
-          ? <div className="ais-range-slider--value">{ Math.round(children * 100) / 100 }</div>
+          ? <div className="ais-range-slider--value">{ Math.round(children) }</div>
```
Comment (iam4x) [inline src/components/Slider/Pit.js]: Why did you changed that?

**Comment by iam4x** [comment]
Nice option I like it 👍

File: src/components/Slider/Pit.js (hunk 1 above)
Comment (Haroenv) [inline src/components/Slider/Pit.js]: That's from the previous PR, I worked on top of it, can fix later

**Comment by Haroenv** [comment]
Regardless of the wrong history, I don't really like how the possible numbers progress, because if you don't pick a number from that first list, it won't show up.

**Comment by bobylito** [comment]
Hey @Haroenv is there something that we can do to make this PR move on? Or should we just close for the time being, until we have a proper plan of attack?

**Comment by Haroenv** [comment]
This solution is flawed because the numbers you can set aren't predictable, so closing

**Pull request closed without merging**